packages/lib-react-components/package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Zooniverse React Components",
   "license": "Apache-2.0",
   "author": "Zooniverse <contact@zooniverse.org> (https://www.zooniverse.org/)",
-  "version": "0.7.2",
+  "version": "1.0.0",
```

Comment:
I think the yarn lock file needs to be updated with this new version of the package.